backend/doc/api.md

```diff
@@ -0,0 +1,188 @@
+base64 encoded in `Authorization` header.
+
+```
+$ http -a admin@admin.admin:admin@admin.admin GET http://127.0.0.1:6543/api/v2/auth/whoami
+HTTP/1.1 200 OK
+Content-Length: 276
+Content-Type: application/json
+Date: Tue, 22 Jan 2019 11:26:53 GMT
+Server: waitress
+Set-Cookie:  session_key=22fd293b6d850faabc8e3f167bcfc804d8713deed03bc15e5029434687050fb809ef2076; expires=Mon, 22-Jan-2019 11:26:53 GMT; Path=/; SameSite=Lax
+
+{
+    "auth_type": "internal", 
+    "avatar_url": null, 
+    "caldav_url": null, 
+    "created": "2019-01-18T13:07:02Z", 
+    "email": "admin@admin.admin", 
+    "is_active": true, 
+    "is_deleted": false, 
+    "lang": null, 
+    "profile": "administrators", 
+    "public_name": "Global manager", 
+    "timezone": "", 
+    "user_id": 1
+}
+```
+
+### Api-Key
+
+you need to set 2 custom header: `Tracim-Api-Key` with a correct api key 
+and `Tracim-Api-Login` with a valid user login.
+
+```
+$ http GET http://127.0.0.1:6543/api/v2/auth/whoami Tracim-Api-Key:changethisnow! Tracim-Api-Login:admin@admin.admin
+HTTP/1.1 200 OK
+Content-Length: 276
+Content-Type: application/json
+Date: Tue, 22 Jan 2019 13:00:49 GMT
+Server: waitress
+Set-Cookie:  session_key=50307c9007fff16791f660eed14d47a33cf11eef365b3e0403ce42f5a8b8f1f12c254b58; expires=Mon, 22-Jan-2018 13:00:49 GMT; Path=/; SameSite=Lax
+
+{
+    "auth_type": "internal", 
+    "avatar_url": null, 
+    "caldav_url": null, 
+    "created": "2019-01-18T13:07:02Z", 
+    "email": "admin@admin.admin", 
+    "is_active": true, 
+    "is_deleted": false, 
+    "lang": null, 
+    "profile": "administrators", 
+    "public_name": "Global manager", 
+    "timezone": "", 
+    "user_id": 1
+}
+```
+
+
+### Login (to get Auth Cookie)
+
+You need to set json properly with `email` and `password` field
+```                                                                                                                                                                                                     
+$ http --session=/tmp/session.json POST  http://127.0.0.1:6543/api/v2/auth/login email=admin@admin.admin password=admin@admin.admin
+HTTP/1.1 200 OK
+Content-Length: 276
+Content-Type: application/json
+Date: Tue, 22 Jan 2019 13:26:03 GMT
+Server: waitress
+Set-Cookie:  session_key=eb61fdb0e496583c7e75a856957ff08b6d66ec1568001dd41d244215a73c1a19de450ca7; expires=Tue, 29-Jan-2019 13:26:03 GMT; Path=/; SameSite=Lax
+
+{
+    "auth_type": "internal", 
+    "avatar_url": null, 
+    "caldav_url": null, 
+    "created": "2019-01-18T13:07:02Z", 
+    "email": "admin@admin.admin", 
+    "is_active": true, 
+    "is_deleted": false, 
+    "lang": null, 
+    "profile": "administrators", 
+    "public_name": "Global manager", 
+    "timezone": "", 
+    "user_id": 1
+}
+
+```
+
+Then you just need to resend cookie using `Cookie` header like any browser do
+automatically to have a temporary access.
+
+```
+$ http --session=/tmp/session.json GET  http://127.0.0.1:6543/api/v2/auth/whoami
```

Comment:
Instead using `--session` parameter, use directly header in `http` command. It will be more excplicit.

Reply:
done.